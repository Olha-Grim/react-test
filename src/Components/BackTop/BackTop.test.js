import React from "react";
import { render } from "@testing-library/react";
import BackTop from ".";

describe("BackTop", () => {
  test("interaction with BackTop component", () => {
    const { asFragment } = render(<BackTop />)
    expect(asFragment(<BackTop />)).toMatchSnapshot();
  });
});
