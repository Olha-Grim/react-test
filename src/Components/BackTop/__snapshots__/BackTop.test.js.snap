// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackTop interaction with BackTop component 1`] = `
<DocumentFragment>
  <div
    class="ant-back-top"
  />
</DocumentFragment>
`;
